spec/spec_helper.rb

```diff
@@ -12,6 +12,6 @@
   config.mock_with :rspec
 
   config.around(:each, :vcr) do |example|
-    VCR.use_cassette(example.metadata[:vcr], re_record_interval: 1.month) { example.call }
+    VCR.use_cassette(example.metadata[:vcr]) { example.call }
```

Comment:
👍 That would seem to be less annoying for developers working on this project. In theory forcing a re-recording keeps us up to date with any signable API changes, but in reality, because this project is so rarely touched, it doesn't really provide us with that assurance.